charts/cheetah-application/values.schema.json

```
@@ -0,0 +1,462 @@
+        "name": {
+          "description": "The name of the service account to use. If not set and create is true, a name is generated using the fullname template",
+          "type": "string"
+        }
+      }
+    },
+    "podLabels": {
+      "description": "Extra pod labels",
+      "type": "object"
+    },
+    "podAnnotations": {
+      "description": "Extra pod annotations",
+      "type": "object"
+    },
+    "containerPort": {
+      "description": "Which container port to use for primary traffic",
+      "type": "number"
+    },
+    "volumes": {
+      "description": "Extra volumes added to the pod. See https://kubernetes.io/docs/concepts/storage/volumes/",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.28.0/volume.json"
+      }
+    },
+    "volumeMounts": {
+      "description": "Extra volume mounts added to the primary container. See https://kubernetes.io/docs/concepts/storage/volumes/",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "$ref": "https://raw.githubusercontent.com/yannh/kubernetes-json-schema/master/v1.28.0/volumemount.json"
+      }
+    },
+    "service": {
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Which type of service to expose the pods with",
+          "type": "string",
+          "enum": [
+            "ClusterIP",
+            "NodePort",
+            "LoadBalancer",
+            "ExternalName",
+            "Headless"
+          ]
+        },
+        "port": {
+          "description": "Which service port to use",
+          "type": "number"
+        }
+      }
+    },
+    "ingress": {
+      "type": "object",
+      "allOf": [
+        {
+          "if": {
+            "properties": {
+              "enabled": {
+                "const": true
+              }
+            }
+          },
+          "then": {
+            "required": ["tls", "hosts", "className"]
+          }
+        }
+      ],
+      "properties": {
+        "enabled": {
+          "description": "Whether to expose the service or not",
+          "type": "boolean"
+        },
+        "className": {
+          "description": "Which ingressClass to use",
+          "type": "string"
+        },
+        "annotations": {
+          "description": "Extra ingress annotations. @default -- (see [values.yaml](values.yaml))",
+          "type": "object"
+        },
+        "hosts": {
+          "description": "Host configuration. See [values.yaml](values.yaml) for formatting",
+          "type": "array",
+          "required": ["host", "paths"],
+          "items": {
+            "type": "object",
+            "properties": {
+              "host": { "type": "string" },
+              "paths": {
+                "type": "array",
+                "items": {
+                  "type": "object",
+                  "required": ["path", "pathType"],
```

I don't _think_ `pathType` is required :thinking: 
```suggestion
                  "required": ["path"],
```